sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerModelFactory.cs

```diff
@@ -0,0 +1,339 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using Azure.AI.FormRecognizer.Training;
+
+namespace Azure.AI.FormRecognizer.Models
```

Comment:
do you think it will be good to have a model factory for training and another for the rest?

Reply:
Discussed offline. Conclusion: mocking is an advanced scenario, and having all models created by a single factory is not a problem since well-experienced users won't have trouble finding them.